src/main/java/edu/ucla/library/avpairtree/AvPtConstants.java

```diff
@@ -21,6 +21,11 @@
      */
     public static final String SYSTEM_TMP_DIR = System.getProperty("java.io.tmpdir");
 
+    /**
+     * The template string for AWS S3 resource URLs. The slots are (in order): bucket name, region, and object key.
+     */
+    public static final String AUDIOWAVEFORM_URL_TEMPLATE = "https://{}.s3-{}.amazonaws.com/{}";
```

Comment:
Here is the URL of the waveform (which the IIIF manifest will point to).

Reply:
My memory is fuzzy, too, but wouldn't this be the new `static.library.ucla.edu` domain now?

Reply:
Yes. Anything with a .dat extension to static.library.ucla.edu will proxy to the audiowave form bucket. I.e:

https://static.library.ucla.edu/arkvalue/file.dat proxies to https://prod-audiowaveform-resources.s3.us-west-2.amazonaws.com/arkvalue/file.dat

Reply:
Ok, thanks. I have changed the app so that we can configure it to use the proxy in prod:

```bash
export AUDIOWAVEFORM_S3_BUCKET=prod-audiowaveform-resources
export AUDIOWAVEFORM_S3_OBJECT_URL_TEMPLATE=https://static.library.ucla.edu/{}
...
```